src/report/report.py

```diff
@@ -539,8 +536,13 @@ def _get_issue_description(
         Returns:
             str: String object representing the description.
         """
-        link_line = f"*Prow Job Link:* [{job_name} #{build_id}|https://prow.ci.openshift.org/view/gs/test-platform-results/logs/{job_name}/{build_id}]"
-        build_id_line = f"*Build ID:* {build_id}"
+        link_line_base_url = (
+            "https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/"
```

Comment:
Is this OK to expose this link?

Reply:
Looks like it is exposed [quite a few times](https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml#L391-L398) in the release repo. That file specifically is owned by DPTP, so I assume it is fine. It is also protected with SSO.